src/backend/joanie/tests/core/api/organizations/test_contracts_signature_link.py

```diff
@@ -226,10 +226,12 @@ def test_api_organization_contracts_signature_link_specified_course_product_rela
         relation = factories.CourseProductRelationFactory(
             organizations=[organization, other_organization],
             product__contract_definition=factories.ContractDefinitionFactory(),
+            product__price=0,
```

Comment:
How can the price be 0 for the product when the payment schedule shows 200.00 in amount ? Is it normal ?

Reply:
Sorry, I missed your message…
This is only a trick to simplify the test.
I'll look for more coherent solution.